### DISCUSSION TITLE
script transform including class from plugins\transforms\excel\lib

### DISCUSSION THREAD

**Discussion by marvinhorst** [apache/hop#4600] [Q&A]
I need to process/input an excel file in a non row by row manner. I thought I'd use a script transform but can't get access to the classes I need which are in the plugins\transforms\excel\lib folder. I was hoping to handle this in the pipeline without having to modify CLASSPATH in the hop executable scripts.

**Comment by bamaer** [comment]
if you can change your hop installation, a dirty hack to get this working is to copy the jars from your plugin's lib folder to `core/lib`. Like I said: dirty hack. Did I mention this is a dirty hack?
